brave/brave-core
# [Brave News]: Add translations for new channels

Resolves https://github.com/brave/brave-browser/issues/27024

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Either run Brave with `--brave-today-host=brave-today-cdn.bravesoftware.com` or normally, once the staging backend has been deployed.
2. Open the Brave News customize dialog
3. Check the dev console - there should be no `Couldn't find translation for channel` messages
